scenes/main_menu/main_menu.tscn

```diff
@@ -184,8 +184,8 @@ tracks/6/keys = {
 
 [sub_resource type="AnimationLibrary" id="AnimationLibrary_nkb72"]
 _data = {
-"RESET": SubResource("2"),
-"path": SubResource("3")
+&"RESET": SubResource("2"),
```

Comment:
Never seen that in tscn before, must be new. I wonder if this is somehow connected with the fact that there were issues with animations.

Reply:
I guess this should wait until testing as well.